js/index.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] Please try to avoid disabling Eslint. It will help to make better code quality.